docs/conf.py

```diff
@@ -49,9 +57,10 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'Tutorials'
```

Comment:
All of this information can be directly pulled from the ``tutorials`` package itself.
This saves on some future maintanance.